doc_src/tutorial.rst

```diff
@@ -422,15 +422,15 @@ fish supports the familiar ``&&`` and ``||`` to combine commands, and ``!`` to n
 
     > ./configure && make && sudo make install
 
-Here, `make` is only executed if `./configure` succeeds (returns 0), and `sudo make install` is only executed if both `./configure` and `make` succeed.
+Here, ``make`` is only executed if ``./configure`` succeeds (returns 0), and ``sudo make install`` is only executed if both ``./configure`` and ``make`` succeed.
 
-fish also supports ``and``, ``or``, and ``not``. The first two are job modifiers and have lower precedence. Example usage::
+fish also supports :ref:`and <cmd-and>`, :ref:`or <cmd-or>`, and :ref:`not <cmd-not>`. The first two are job modifiers and have lower precedence. Example usage::
 
     > cp file1.txt file1_bak.txt && cp file2.txt file2_bak.txt ; and echo "Backup successful"; or echo "Backup failed"
     Backup failed
 
 
-As mentioned in `the section on the semicolon <#tut_semicolon>`__, this can also be written in multiple lines, like so::
+As mentioned in the section on the semicolon, this can also be written in multiple lines, like so::
```

Comment:
This (and combiners) should still be linked, no?

Or am I missing something?

Reply:
Is it possible to link directly to a section? I tried a few things. For example the HTML link ends "#separating-commands-semicolon", but the following didn't work:

```
:ref:`the section on the semicolon <separating-commands-semicolon>`
```

Is the best way is to add a label above each section?

```
.. _separating-commands-semicolon:

Separating Commands (Semicolon)
-------------------------------
```


Also came across the [autosection](https://www.sphinx-doc.org/en/master/usage/extensions/autosectionlabel.html) extension: https://stackoverflow.com/a/40225340/2562199

Reply:
Yeah, just add the label. I'd call it "semicolon" here.

Reply:
Done.